i18n/en.yaml

```diff
@@ -1,4 +1,4 @@
-not-found:
+unot-found:
```

Comment:
Was this an intentional change?

Reply:
Oohh def a typo!!! Thanks for catching!!!